### PR TITLE
ONL-4821 upgrade sb to v6

### PR DESCRIPTION
**Pull request by jaimeperezpedra**
**PR to upgrade sb to v6**

Things did:

- Revome all the export stories from the story.js
- Changed the way to set the backgrounds for the stories [the new way](https://storybook.js.org/docs/react/essentials/backgrounds)
- `config.js` is now `preview.js`
- `main.js` is where you put all the `webpack.config` and also where you load the `stories` and the `addons`.
- `manager.js` to set the `theme` [link](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#using-managerjs)
- A few changes on the `visual-regression.spec.js` to fix the changes from **SB**
- For now the parameters on the stories will be after adding the story to the `stories`, `addParameters` it's not working on the visual regression test, example:

> stories
> .add('basic', () => ({
> .
> .
> .
> }), {
>   visualRegressionTests: { enabled: false },
>});

### CODE REVIEW

**Review by matallo — APPROVED** [review]

**Review by mcibique — COMMENTED** [review]
There is a warning about vue-docgen-loader not being able to read ec-smart-table definition. It might be caused because smart-table is a HoC so it cannot be resolved during the build time. can we at least suppress the warning in the console?

File: package.json [hunk 1]
```diff
@@ -78,6 +80,7 @@
     "postcss-mixins": "6.2.3",
     "postcss-nested": "4.2.3",
     "postcss-preset-env": "6.7.0",
+    "react-is": "16.13.1",
```
[the remark this answers is not in the document]
Comment (jaimeperezpedra) [reply package.json:83]: Removed

File: package.json [hunk 2]
```diff
@@ -41,16 +42,17 @@
   },
   "devDependencies": {
     "@ascendancyy/vue-cli-plugin-stylelint": "1.1.2",
+    "@babel/core": "7.11.6",
```
[the remark this answers is not in the document]
Comment (jaimeperezpedra) [reply package.json:45]: Removed

File: package.json [hunk 3]
```diff
@@ -32,6 +32,7 @@
     "release": "npm version patch && npm publish && git push --follow-tags"
   },
   "dependencies": {
+    "@storybook/cli": "6.0.21",
```
[the remark this answers is not in the document]
Comment (jaimeperezpedra) [reply package.json:35]: Removed

**Comment by jaimeperezpedra** [comment]
@mcibique I didn't find anything of this warning...maybe idk how to search that, but in prod this is not showing at all.

**Pull request merged**